scripts/rspec/rspec-templates/VbNetTestCaseTemplate.vb

```diff
@@ -7,6 +7,6 @@ Namespace Tests.TestCases
     Class Foo
         Public Sub Test()
 
-		End Sub
+        End Sub
     End Class
 End Namespace
```

Comment:
For VB, there should be more empty lines around this. I also changed the class name to `Program` to be same as C#
```
Namespace Tests.TestCases

    Class Program

        Public Sub Test()

        End Sub

    End Class

End Namespace
```